deflector:  # set the namespace to be used in parameter server
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: prismatic_joint1
    pid: {p: 500.0, i: 0.0, d: 500.0}
  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: prismatic_joint2
    pid: {p: 50000.0, i: 1000.0, d: 1000.0}
  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: prismatic_joint3
    pid: {p: 600.0, i: 0.01, d: 150.0}
